mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -630,47 +646,46 @@ impl JsonRpcClient for NativeClientImpl {
     fn client_info(&self) -> String { UtxoJsonRpcClientInfo::client_info(self) }
 
     #[cfg(target_arch = "wasm32")]
-    fn transport(&self, _request: JsonRpcRequestEnum) -> JsonRpcResponseFut {
-        Box::new(futures01::future::err(ERRL!(
-            "'NativeClientImpl' must be used in native mode only"
+    fn transport(&self, _request: JsonRpcRequestEnum) -> JsonRpcTransportErrFut {
+        Box::new(futures01::future::err(JsonRpcTransportError::NativeClientImplError(
+            "'NativeClientImpl' must be used in native mode only".to_string(),
         )))
     }
 
     #[cfg(not(target_arch = "wasm32"))]
-    fn transport(&self, request: JsonRpcRequestEnum) -> JsonRpcResponseFut {
+    fn transport(&self, request: JsonRpcRequestEnum) -> JsonRpcTransportErrFut {
```

Comment:
This should be `JsonRpcResponseFut` like before

Reply:
done